protocol/protocol.tex

```diff
@@ -12074,6 +12074,12 @@ \subsubsection{BLAKE2s hashes} \label{cctblake2s}
         While it might be possible to use variants of functions with low circuit cost
         such as MiMC \cite{AGRRT2017}, it was felt that they had not yet received
         sufficient cryptanalytic attention to confidently use them for \Sapling.
+  \item The actual number of constraints for $\BlakeTwosGeneric$ in the \Sapling
+        circuit is $4 \mult 2 \mult 32 = 256$ fewer than what is calculated
+        above, for a total of 21006 constraints. This is because $v$ is constant
+        at the start of the first round, so in the first four calls to $G$, the
+        parameters $b$ and $d$ are constant, eliminating the constraints for the
+        first two XORs in those four calls to $G$.
```

Comment:
This looks correct. I would it this in the main text rather than the non-normative notes. It also needs to be taken into account in the table in appendix A.4.